spec/rspec/matchers/dsl_spec.rb

```diff
@@ -591,11 +591,11 @@ def foo
         end
 
         specify "`match?` returns true if the wrapped expectation passes" do
-          expect(self.matcher.matches?('value')).to be_truthy
+          expect(matcher.matches?('value')).to be_truthy
         end
 
         specify "`match?` returns false if the wrapped expectation fails" do
-          expect(self.matcher.matches?('other value')).to be_falsey
+          expect(matcher.matches?('other value')).to be_falsey
```

Comment:
Again I feel `self.` is more suitable, can we supress the rule instead

Reply:
Sorry, not exactly sure what you mean. Is `matcher` being referred to in `self.matcher` the one defined in a `let`?

Reply:
No, not sure what I meant here.